src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java

```diff
@@ -157,7 +158,11 @@ private static Node createFileDisplay(LinkedFileViewModel linkedFile) {
         Button acceptAutoLinkedFile = IconTheme.JabRefIcons.AUTO_LINKED_FILE.asButton();
         acceptAutoLinkedFile.setTooltip(new Tooltip(Localization.lang("This file was found automatically. Do you want to link it to this entry?")));
         acceptAutoLinkedFile.visibleProperty().bind(linkedFile.isAutomaticallyFoundProperty());
-        acceptAutoLinkedFile.setOnAction(event -> linkedFile.acceptAsLinked());
+        acceptAutoLinkedFile.managedProperty().bind(linkedFile.isAutomaticallyFoundProperty());
+        acceptAutoLinkedFile.setOnAction(event -> {
+            linkedFile.acceptAsLinked();
+            linkedFile.setOpacityProperty(1.0);
```

Comment:
Move `linkedFile.setOpacityProperty(1.0);` into `acceptAsLinked`. The viewmodel should contain all the logic.